setup.py

```diff
@@ -9,6 +10,21 @@
     changes = f.read()
 
 
+install_requires = [
+    'pytest',
+    'pep8',
+    'pyflakes',
+    'coverage',
+    'sphinx',
+    'pytest-cov',
```

Comment:
This isn't strictly true.

We've written `uflash` to NOT DEPEND on anything other than Python's standard library. That's why I put these things in the requirements.txt file because that's what developers would know how to use.

It also means that downstream packagers (e.g. for Raspbian) who build their .debs from setup.py will be creating unnecessary dependencies.

Can we move these back to requirements.txt please..? (sorry - happy to be argued with if you disagree) :-)